packages/experimental/NativeDatePicker/ios/DatePickerManager.swift

```diff
@@ -0,0 +1,66 @@
+import FluentUI
+
+@objc(MSFDatePickerManager)
+public class DatePickerManager: NSObject {
+    var lastDelegate: DatePickerDelegate?
+
+    @objc public class func requiresMainQueueSetup() -> Bool {
+        return true
+    }
+
+    @objc public func present(
+        mode: DateTimePickerMode,
+        dateRangePresentation: DateTimePicker.DateRangePresentation,
+        datePickerType: DateTimePicker.DatePickerType,
+        startDate: Date?,
+        endDate: Date?,
+        startTitle: String?,
+        startSubtitle: String?,
+        startTab: String?,
+        endTitle: String?,
+        endSubtitle: String?,
+        endTab: String?,
+        dateTitle: String?,
+        dateSubtitle: String?,
+        timeTitle: String?,
+        timeSubtitle: String?,
+        callback: @escaping RCTResponseSenderBlock
+    ) {
+        DispatchQueue.main.async {
+            guard let viewController = UIApplication.shared.windows.filter({$0.isKeyWindow}).first?.rootViewController else {
+                fatalError("Unable to get a UIViewController from current shared application context.")
+            }
```

Comment:
Maybe add a comment to the stack overflow you got this from / why you need to do this?

Reply:
Added in `30c439e`.  I didn't see any other FURN code cite stackoverflow, so I just added some explanatory commentary.